book/src/paper-wallet/installation.md

```diff
@@ -8,30 +8,36 @@ After installation, ensure your version is `0.21.1` or higher by running `solana
 First, download the desired release tarball from GitHub. The examples below will
 retrieve the most recent release. If you would like to download a specific
 version instead replace `latest/download` with `download/VERSION` where VERSION
-is a tag name from https://github.com/solana-labs/solana/releases (ie. v0.21.1).
+is a tag name from https://github.com/solana-labs/solana/releases (e.g. v0.21.1).
```

Comment:
@t-nelson I think we can remove the "specific version" instructions. Did you have a specific scenario in mind for someone wanting a certain version?

Reply:
I was thinking it'd reduce support requests a bit in the event we have a buggy release.  The goal was to give someone unfamiliar with the interface some guidance as to which string is a "tag".  I don't think we need to keep the version pointed at anything, nor will I be hurt if you drop it 🙂

Reply:
Yeah that's a good point, I think we could react to that scenario pretty quickly if it came up though. I'd like to stick with dropping it